Cadence/index.ts

```diff
@@ -44,7 +44,9 @@ export namespace Cadence {
 			else
 				result = cadence.value
 		}
-		return options?.cap == undefined ? result : Math.min(options.cap, result)
+		return options?.limit == undefined
+			? result
+			: Math.trunc(Math.min(options.limit, result) / cadence.value) * cadence.value
```

Comment:
why do we divide by cadence.value, truncate the result and then multiply with cadence.value again?

Reply:
comment for line 41 and 43:  Do we handle year stuff  when the cadence is 'week' and 'day'?

Reply:
magic

Reply:
seems reasonable